AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Core/ListTests.swift

```diff
@@ -102,58 +102,6 @@ class ListTests: BaseDataStoreTests {
         wait(for: [expect], timeout: 1)
     }
 
-    /// - Given: a list a `Post` and a few comments associated with it
```

Comment:
Why are these test removed? Should we add similar test with the non-deprecated api?

Reply:
i believe we renamed load to fetch mainly.. and also completely removed `loadAsPublisher()`  so that is why this can completely be removed

Reply:
Yes, there are success and failure tests for fetch API here : 
https://github.com/aws-amplify/amplify-ios/blob/dev-preview/AmplifyTests/CategoryTests/DataStore/Model/ListPaginationTests.swift

Reply:
Yup, fetch() is asynchronous api and this test was part of the synchronous load() API which was removed. 
There are tests for fetch api here : 
https://github.com/aws-amplify/amplify-ios/blob/dev-preview/AmplifyTests/CategoryTests/DataStore/Model/ListPaginationTests.swift